LayoutTests/fast/scrolling/rtl-bt-drag-scroller.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+</head>
+<body style="margin: 0px;">
+<div id="scrollMe" style="writing-mode: vertical-rl; direction: rtl; width: 100px; height: 100px; position: absolute; left: 0px; top: 0px; overflow: scroll;">
+<div style="width: 300px; height: 300px; background-color: yellow;"></div>
+</div>
+<script>
+description("This test makes sure that a div with right-to-left or bottom-to-top scroll directions can be scrolled by dragging the scrollbar.");
+
+if (window.eventSender) {
+    eventSender.mouseMoveTo(96, 96);
+    eventSender.mouseDown();
```

Comment:
I think this will fail on `iOS`. @pxlcoder - Does the test need to be using 'UIHelper' to make it work on 'iOS' with 'scrollTo' usage?

Reply:
I've added a expectation for failure on iOS. Meanwhile, I found this issue reproducible on WebKitGTK as well so it's not Mac-only.